lib/kitchen/driver/oci/instance/compute.rb

```diff
@@ -81,6 +81,21 @@ def instance_source_via_image
             )
           end
 
+          # Adds the instance options property to the launch details.
+          def instance_options
+            return if config[:instance_options].empty?
+
+            instance_options = {}
+            config[:instance_options].each do |k, v|
+              case k
+              when :are_legacy_imds_endpoints_disabled
+                instance_options[:are_legacy_imds_endpoints_disabled] = v
+              end
+            end
```

Comment:
I think I would like to see this be a little bit more flexible in terms of not building in future revisions to the driver if the upstream API begins accepting additional options. I'd like to see the driver pass the `instance_options` hash provided by the user to the upstream model and specify in explicit terms in the README that there is at this point one valid key:value pair that is accepted by the model.